frontend_lib/src/component/ShareLink/ShareLink.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { translate } from 'react-i18next'
+import Radium from 'radium'
+import ComposedIcon from '../Icon/ComposedIcon.jsx'
+
+class ShareLink extends React.Component {
+  handleCopyToClipboard = () => {
+    let tmp = document.createElement('textarea')
+    document.body.appendChild(tmp)
+    tmp.value = this.props.link
+    tmp.select()
+    document.execCommand('copy')
+    document.body.removeChild(tmp)
+    /* or https://stackoverflow.com/questions/36639681/how-to-copy-text-from-a-div-to-clipboard See what's best */
+  }
+
+  render () {
+    const { props } = this
+
+    return (
+      <div className='shareLink d-flex'>
+        <div className='shareLink__icon'>
+          <ComposedIcon
+            icon='link'
+            smallIcon='lock'
+            style={{color: '#ababab'}}
+            smallIconStyle={{color: '#252525'}} // See how to use Variable's variables
```

Comment:
add the comment convention and add the file's extension of Variable.styl to make it easier to understand